components/org.wso2.carbon.identity.organization.management.service/src/test/java/org/wso2/carbon/identity/organization/management/service/dao/impl/OrganizationManagementDAOImplTest.java

```diff
@@ -0,0 +1,248 @@
+import static org.wso2.carbon.identity.organization.management.util.TestUtils.mockDataSource;
+
+public class OrganizationManagementDAOImplTest {
+
+    private OrganizationManagementDAO organizationManagementDAO = new OrganizationManagementDAOImpl();
+    private static final Calendar CALENDAR = Calendar.getInstance(TimeZone.getTimeZone(UTC));
+    private static final String ATTRIBUTE_KEY = "country";
+    private static final String ATTRIBUTE_VALUE = "Sri Lanka";
+    private static final String ORG_NAME = "XYZ builders";
+    private static final String INVALID_DATA = "invalid data";
+    private static final String SUPER_ORG_ID = "10084a8d-113f-4211-a0d5-efe36b082211";
+    private String orgId;
+
+    @BeforeClass
+    public void setUp() throws Exception {
+
+        initiateH2Base();
+        mockDataSource();
+        storeChildOrganization(SUPER_ORG_ID);
+    }
+
+    @AfterClass
+    public void tearDown() throws Exception {
+
+        closeH2Base();
+    }
+
+    @Test
+    public void testAddOrganization() throws Exception {
+
+        String orgId = generateUniqueID();
+
+        Organization organization = new Organization();
+        organization.setId(orgId);
+        organization.setName("org1");
+        organization.setDescription("org1 description.");
+        organization.setCreated(Instant.now());
+        organization.setLastModified(Instant.now());
+        organization.setStatus(OrganizationManagementConstants.OrganizationStatus.ACTIVE.toString());
+        organization.setType(STRUCTURAL.toString());
+
+        ParentOrganizationDO parentOrganizationDO = new ParentOrganizationDO();
+        parentOrganizationDO.setId(SUPER_ORG_ID);
+        organization.setParent(parentOrganizationDO);
+
+        List<OrganizationAttribute> attributes = new ArrayList<>();
+        attributes.add(new OrganizationAttribute(ATTRIBUTE_KEY, ATTRIBUTE_VALUE));
+        organization.setAttributes(attributes);
+
+        organizationManagementDAO.addOrganization(organization);
+        Assert.assertNotNull(organizationManagementDAO.getOrganization(orgId));
+    }
+
+    @DataProvider(name = "dataForIsOrganizationExistByName")
+    public Object[][] dataForIsOrganizationExistByName() {
+
+        return new Object[][]{
+
+                {ORG_NAME},
+                {INVALID_DATA},
+        };
+    }
+
+    @Test(dataProvider = "dataForIsOrganizationExistByName")
+    public void testIsOrganizationExistByName(String name) throws Exception {
+
+        boolean organizationExistByName = organizationManagementDAO.isOrganizationExistByName(name);
+        if (StringUtils.equals(name, ORG_NAME)) {
+            Assert.assertTrue(organizationExistByName);
+        } else if (StringUtils.equals(name, INVALID_DATA)) {
+            Assert.assertFalse(organizationExistByName);
+        }
+    }
+
+    @DataProvider(name = "dataForIsOrganizationExistById")
+    public Object[][] dataForIsOrganizationExistById() {
+
+        return new Object[][]{
+
```

Comment:
Shall we remove unnecessary new line. Check similar places

Reply:
resolved by 6534bc4a65f7a5c7396abda24169fb0e5690702a